pkg/filter/stream/ipaccess/stream_filter_test.go

```diff
@@ -265,6 +266,14 @@ func (m *MockHost) SupportTLS() bool {
 	return false
 }
 
+func (m *MockHost) StartTime() time.Time {
```

Comment:
should use tool auto generate code, look this package `readme.md`

Reply:
> should use tool auto generate code, look this package `readme.md`

Sorry, I'm not found related codegen tools :(

Reply:
https://github.com/mosn/mosn/blob/master/pkg/mock/readme.md